src/controllers/commentController.js

```diff
@@ -20,11 +22,33 @@ export default class CommentController {
       const { user } = req;
       const { comment } = req.body;
       const { requestId } = req.params;
+      const { email } = req.managerDetails;
       const newComment = await db.Comments.create({
         requestId,
         commmentOwner: user.id,
         comment,
       });
+      if (user.role === 'manager') {
+        const notifyUser = await notifService.createNotif(req.request.userId, req.request.email, 'your manager posted a comment', '#');
+        const content = {
+          intro: req.__(notifyUser.content),
+          instruction: req.__(`To view this ${newComment.comment} click below`),
+          text: req.__('View comment'),
+          signature: req.__('signature')
+        };
+        // await
+        SendNotification.SendNotif(notifyUser, req, content);
```

Comment:
hey @bbaime98 could you change the SendNotif function to lowercase because it's not a class

Reply:
okay let me do it.

Reply:
@izzett222  it has been implemented.